core/src/test/resources/META-INF/services/org.apache.spark.scheduler.ExternalClusterManager

```diff
@@ -1 +1,2 @@
-org.apache.spark.scheduler.DummyExternalClusterManager
\ No newline at end of file
+org.apache.spark.scheduler.DummyExternalClusterManager
+org.apache.spark.scheduler.MockExternalClusterManager
```

Comment:
Did you ever look at combining DummyExternalClusterManager and MockExternalClusterManager?  They are just two variations on a fake ExternalClusterManager for use in tests.  I realize that the focus of the tests for Dummy... and Mock... are different, so the two variations may not be easy or clean to combine, but if we could have just one fake ExternalClusterManager that still had a relatively clean implementation, I think that would be better than maintaining two.  OTOH, if combining them gets messy, then just go with what you've already got.